src/Components/CreateImageWizard/formComponents/ReviewStepTextLists.js

```diff
@@ -0,0 +1,514 @@
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Account type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {props.googleAccType?.[getState()?.values?.['google-account-type']]}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          {props.googleAccType?.[
+            getState()?.values?.['google-account-type']
+          ] === 'Domain'
+            ? 'Domain'
+            : 'Email address'}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['google-email'] ||
+            getState()?.values?.['google-domain']}
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvAzureList = () => {
+  const { getState } = useFormApi();
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>Microsoft Azure</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Azure Tenant ID
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['azure-tenant-id']}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Subscription ID
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['azure-subscription-id']}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Resource group
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['azure-resource-group']}
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvOtherList = () => {
+  return (
+    <>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Built image will be available for download
+        </TextListItem>
+      </TextList>
+      <br />
+    </>
+  );
+};
+
+export const FSCList = ({ ...props }) => {
```

Comment:
I think here we can eliminate {...props} entirely if we just move minSize state into this component, but in general, would you be okay with _not_ using prop destructuring? 

We don't have a style guide (maybe we should work on that, or at least get an eslint plugin) but I have started to really prefer explicitly declaring props whenever possible, in my opinion it reduces the cognitive load and makes it easier to understand components. With {...props} I am not sure what is actually getting passed in, plus it also makes it possible to pass in un-used props that cause additional re-renders for no good reason (inefficient). I'd like to use {...props} only when absolutely necessary. 

That's my personal preference but maybe there is also a good reason for using the destructuring I'm not aware of... what do you think?